indico/modules/events/abstracts/templates/_abstract/_common.html

```diff
@@ -0,0 +1,12 @@
+{% macro render_edited_hint(dt, user=None) %}
```

Comment:
But there are non-underscored files inside. It's true that maybe underscoring the dir is a bit too much.